mozilla-release/mobile/android/base/locales/en-US/android_strings.dtd

```diff
@@ -597,7 +597,9 @@
 
 <!-- Localization note (home_title): This string should be kept in sync
      with the page title defined in aboutHome.dtd -->
-<!ENTITY home_title "&brandShortName; Home">
+<!-- Cliqz start -->
+<!ENTITY home_title "Start Tab">
```

Comment:
Don't modify strings in android_strings.dtd 
Add a new string in cliqz_strings.dtd (Thats the file which we send to translators)